components/resizable/doc/index.en-US.md

```diff
@@ -0,0 +1,85 @@
+---
+category: Components
+type: Experimental
+title: Resizable
+cols: 1
+---
+
```

Comment:
Add an alert there telling users that this component is experimental and we don't ensure the stability of the API.

We can add this alert in the site generating script.